airflow/providers/apache/druid/hooks/druid.py

```diff
@@ -136,7 +136,7 @@ class DruidDbApiHook(DbApiHook):
     For ingestion, please use druidHook.
     """
 
-    conn_name_attr = 'druid_broker_conn_id'
+    conn_name_attr = 'conn_id'
```

Comment:
Can you explain this change a bit

Reply:
I don't think this is a wrong connection name -- we use the same everywhere for other hooks too

Reply:
Right, I close that one and take a closer look.